mava/logger.py

```diff
@@ -13,136 +13,153 @@
 # limitations under the License.
 
 """Logger setup."""
-from typing import Dict, Optional, Protocol
+from typing import Dict, Optional, Tuple, Union
 
 import jax.numpy as jnp
-import numpy as np
 from colorama import Fore, Style
 
 from mava.types import ExperimentOutput
-from mava.utils.logger_tools import Logger
+from mava.utils.logger_tools import Logger as LoggerTools
```

Comment:
This is a bit confusing can we clean this up a bit more and call it a `MultiLogger` (the class and the file)